app/assets/stylesheets/custom.scss

```diff
@@ -1,20 +1,5 @@
-<<<<<<< HEAD
-=======
-
->>>>>>> b32c8d64aadb4eb43d1b526bf3da7229dcf40197
 @import "bootstrap";
```

Comment:
You can remove this or remove it from application.scss, you don't need them in both places.